packages/abi-coder/src/interface.ts

```diff
@@ -42,13 +42,15 @@ export default class Interface {
   readonly abi: ABI | null;
   readonly types: ReadonlyArray<JsonFlatAbiFragmentType>;
   readonly loggedTypes: ReadonlyArray<JsonAbiLogFragment>;
+  private logTypeHash: { [id: string]: ReadonlyArray<JsonAbiLogFragment> };
```

Comment:
This variable name could be more self-explanatory.

Here's a simple suggestion to increase clarity.

```ts
/*
  Same as the `loggedTypes` above, but dedicated to external contracts
  added via `<base-invocation-scope.ts>.addContracts()` method. This is
  used to decode logs from contracts other than the main contract
  we're interacting with.
*/
private externalLoggedTypes: { [contractId: string]: ReadonlyArray<JsonAbiLogFragment> };
```